src/frontend/components/Dashboard/index.spec.tsx

```diff
@@ -1,18 +1,20 @@
 import { render, screen } from '@testing-library/react';
-import React, { Suspense } from 'react';
+import React, { lazy, Suspense } from 'react';
 
 import { Document } from '../../types/file';
-import { modelName } from '../../types/models';
 import { uploadState, Video } from '../../types/tracks';
 import {
   documentMockFactory,
   videoMockFactory,
 } from '../../utils/tests/factories';
 import { wrapInIntlProvider } from '../../utils/tests/intl';
 import { Loader } from '../Loader';
-import Dashboard from './index';
 import { wrapInRouter } from '../../utils/tests/router';
 
+const Dashboard = lazy(() => import('../Dashboard'));
+const DashboardDocument = lazy(() => import('../DashboardDocument'));
+const DashboardVideo = lazy(() => import('../DashboardVideo'));
+
```

Comment:
I don't understand why you are lazily import those files even more you mocked them.
I did some test from my side and it appears that the problem about `act` is related to this point.

Reply:
Oh great, thank you!
This was a suggestion, that worked well with jest 26, and I didn't really think about it :/
I just blindly applied it.